core/src/main/java/org/apache/calcite/rel/RelDistributions.java

```diff
@@ -38,19 +38,14 @@
 public class RelDistributions {
   public static final ImmutableIntList EMPTY = ImmutableIntList.of();
 
-  /** The singleton singleton distribution. */
+  /** The singleton distribution. */
   public static final RelDistribution SINGLETON =
       new RelDistributionImpl(RelDistribution.Type.SINGLETON, EMPTY);
 
   /** The singleton random distribution. */
   public static final RelDistribution RANDOM_DISTRIBUTED =
       new RelDistributionImpl(RelDistribution.Type.RANDOM_DISTRIBUTED, EMPTY);
 
-  /** The singleton round-robin distribution. */
-  public static final RelDistribution ROUND_ROBIN_DISTRIBUTED =
```

Comment:
Why are you proposing to remove this? (Notice that, Calcite is a framework used by many third-party projects, hence even some code has no usage in Calcite itself, we cannot remove them safely)

Reply:
ok，I got it. I would close the pr，thanks~